server/api/settings.py

```diff
@@ -176,6 +177,6 @@
 
 # DRF Simple JWT Configuration
 SIMPLE_JWT = {
-    # "ACCESS_TOKEN_LIFETIME": datetime.timedelta(seconds=10),
-    # "REFRESH_TOKEN_LIFETIME": datetime.timedelta(seconds=30),
-}
+    "ACCESS_TOKEN_LIFETIME": datetime.timedelta(seconds=10),
+    "REFRESH_TOKEN_LIFETIME": datetime.timedelta(seconds=30),
```

Comment:
Remove these lines as it will cause user session to expire in 30 seconds. This is just a remnant from when I was trying to fix the token refresh.